### PR TITLE
Move definitions to VTKBase.jl

### PR DESCRIPTION
**Pull request by jipolanco**
See discussion in #23

We need to wait until VTKBase.jl is registered before merging.

### CODE REVIEW

**Comment by fredrikekre** [comment]
FYI, I ran https://github.com/Ferrite-FEM/Ferrite.jl tests with this PR and everything passed since the VTKCellTypes module is still exported after this PR, so `using WriteVTK` followed by e.g. https://github.com/Ferrite-FEM/Ferrite.jl/blob/b019eea16916f1e0895d675954ed572396c9ac2a/src/Export/VTK.jl#L1 still works.

**Comment by jipolanco** [comment]
@fredrikekre Great, thanks for checking. BTW we have just moved WriteVTK.jl and VTKBase.jl to a new GH organisation. If you're interested we can add you as a member.

**Comment by fredrikekre** [comment]
Sure, thanks.

File: Project.toml [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
 LightXML = "9c8b4983-aa76-5018-a973-4c85ecc9e179"
 TranscodingStreams = "3bb67fe8-82b1-5028-8e26-92a6c54297fa"
+VTKBase = "4004b06d-e244-455f-a6ce-a5f9919cc534"
 
 [compat]
 CodecZlib = "0.7"
 FillArrays = "0.13"
 LightXML = "0.9"
 TranscodingStreams = "0.9"
+VTKBase = "0.1"
```
Comment (fredrikekre) [inline Project.toml:19]: Can I suggest releasing 1.0.0 instead? It is more flexible to work with 3 numbers instead of just 2.

Comment (jipolanco) [reply Project.toml:19]: Yes, good idea.

**Pull request merged**